[BUG] bot连接后发生错误 Error when running WebSocketConnection hook

### Python 版本

3.12

### SuggarChat 版本

2.2.0post1

### 发生了什么?

bot连接后初始化失败，未能生成配置文件。
我看了一下代码，config.py 的第 250 行，变量 prompt，但是这个变量在使用之前没有声明。
我尝试在其前面添加`prompt = ""`，问题解决了。

### 如何复现

正常启动，bot连接，然后报错

### 期望的结果?

应该正常生成配置文件

### SuggarChat 运行日志

```shell
04-05 10:28:20 [ERROR] nonebot | Error when running WebSocketConnection hook:
Traceback (most recent call last):
  File "<string>", line 17, in <module>
  File "/workspaces/.venv/lib/python3.12/site-packages/nonebot/__init__.py", line 337, in run
    get_driver().run(*args, **kwargs)
  File "/workspaces/.venv/lib/python3.12/site-packages/nonebot/drivers/fastapi.py", line 187, in run
    uvicorn.run(
  File "/workspaces/.venv/lib/python3.12/site-packages/uvicorn/main.py", line 579, in run
    server.run()
  File "/workspaces/.venv/lib/python3.12/site-packages/uvicorn/server.py", line 66, in run
    return asyncio.run(self.serve(sockets=sockets))
  File "/usr/local/lib/python3.12/asyncio/runners.py", line 194, in run
    return runner.run(main)
  File "/usr/local/lib/python3.12/asyncio/runners.py", line 118, in run
    return self._loop.run_until_complete(task)
> File "/workspaces/.venv/lib/python3.12/site-packages/nonebot/utils.py", line 238, in run_coro_with_catch
    return await coro
  File "/workspaces/.venv/lib/python3.12/site-packages/nonebot/dependencies/__init__.py", line 113, in __call__
    return await cast(Callable[..., Awaitable[R]], self.call)(**values)
  File "/workspaces/plugins/nonebot_plugin_suggarchat/connection.py", line 16, in onConnect
    config_manager.load(bot.self_id)
  File "/workspaces/plugins/nonebot_plugin_suggarchat/config.py", line 250, in load
    f.write(prompt)
UnboundLocalError: cannot access local variable 'prompt' where it is not associated with a value
```

--- a/src/nonebot_plugin_suggarchat/config.py
+++ b/src/nonebot_plugin_suggarchat/config.py
@@ -239,6 +239,7 @@ class ConfigManager:
             self.config.save_to_toml(self.toml_config)
 
         # private_train
+        prompt = ""
         if self.private_prompt.is_file():
             with self.private_prompt.open("r", encoding="utf-8") as f:
                 prompt = f.read()
@@ -250,6 +251,7 @@ class ConfigManager:
                 f.write(prompt)
 
         # group_train
+        prompt = ""
         if self.group_prompt.is_file():
             with self.group_prompt.open("r", encoding="utf-8") as f:
                 prompt = f.read()

